### PR TITLE
Update changelog for upcoming release.

### PR DESCRIPTION
**Pull request by mikelehen**

### CODE REVIEW

File: packages/firestore/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
 # Unreleased
+- [changed] Network streams are automatically closed after 60 seconds of
+  idleness.
+- [changed] We no longer log 'RPC failed' messages for expected failures.
 
 # v0.1.2
 - [fixed] Fixed validation of nested arrays to allow indirect nesting
```
Comment (schmidt-sebastian) [inline packages/firestore/CHANGELOG.md]: - [changed] We now support field deletes for set() with `{merge:true}`.

File: packages/firestore/CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,7 @@
 # Unreleased
```
Comment (jshcrowthe) [inline packages/firestore/CHANGELOG.md]: If we can add a `# v0.1.4` here that'd be great

Comment (mikelehen) [reply packages/firestore/CHANGELOG.md]: done

File: packages/firestore/CHANGELOG.md (hunk 1 above)
Comment (mikelehen) [reply packages/firestore/CHANGELOG.md]: Added:
- [changed] We now support `FieldValue.delete()` sentinels in `set()` calls
  with `{merge:true}`.

**Pull request merged**